src/brevitas_examples/llm/llm_quant/prepare_for_quantize.py

```diff
@@ -2,11 +2,16 @@
 
 import torch
 from transformers.models.opt.modeling_opt import OPTAttention
+from transformers.models.opt.modeling_opt import OPTSdpaAttention
 
 from brevitas.graph import ModuleToModuleByClass
 from brevitas_examples.llm.llm_quant.mha_layers import QuantizableOPTAttention
 
-QUANTIZABLE_MHA_MAP = {OPTAttention: (QuantizableOPTAttention, {'batch_first': True})}
+QUANTIZABLE_MHA_MAP = {
+    OPTAttention: (QuantizableOPTAttention, {
+        'batch_first': True}),
+    OPTSdpaAttention: (QuantizableOPTAttention, {
+        'batch_first': True}),}
```

Comment:
An alternative is to add the argument `attn_implementation="eager"` to the `from_pretrained` call. @Giuseppe5, do you have a preference here? We would need to add a flag to main I guess...